README.md

```diff
@@ -1,6 +1,32 @@
 # Ghidra, batteries included
 
 A distribution of Ghidra 9.2 (as of now unreleased) with extensions and patches included to simplify reversing code patterns commonly found in games.
+
+## **⚠ WARNING: Unstable patches and features included.**  
+
+Before using this Ghidra distribution, you should be aware of the risks of using a version of Ghidra that is not yet considered stable.
+
+### What does _stable_ mean in the context of this distribution?
+
+There are 3 main components to consider when we think about what is _stable_ here:
+
+- Ghidra patches (typically from in-progress pull requests)
+- Ghidra extensions (3rd party plugins like cpp-class-analyzer)
+- Ghidra core
+
+Since we build from the latest versions of these components, we run the risk of introducing backwards compatible changes when Ghidra, a patch, or an extension changes its schema.
```

Comment:
```suggestion
Since we build from the latest versions of these components, we run the risk of introducing non backwards compatible changes when Ghidra, a patch, or an extension changes its schema.
```